### PR TITLE
Improve UX

### PR DESCRIPTION
**Pull request by planeiii-te**
Added a few quality of life improvements:

* Auto-focus room name on render
* Allow `enter` to submit the form

### CODE REVIEW

File: client/pages/play/_roomName.vue [hunk 1]
```diff
@@ -96,6 +99,9 @@ export default {
     },
   },
   methods: {
+    focusInput() {
+      this.$refs.roomNameInput.focus();
+    },
```
Comment (FLamparski) [inline client/pages/play/_roomName.vue:104]: I think you can achieve the same result by putting an `autofocus` attribute on the input, did you try that?

**Review by FLamparski — COMMENTED** [review]
Thanks for the PR! I just have one question

File: client/pages/play/_roomName.vue (hunk 1 above)
Comment (planeiii-te) [reply client/pages/play/_roomName.vue:104]: I did not. It may work, but I'm not sure how the browser will handle the view being shown and hidden.

**Comment by planeiii-te** [comment]
@FLamparski Anything else?

**Review by FLamparski — APPROVED** [review]

**Pull request merged**